app/webpack/javascripts/channels/survey_result_channel.js

```diff
@@ -2,19 +2,19 @@ import consumer from "./consumer"
 import { chart } from "../chart";
 
 consumer.subscriptions.create({ channel: "SurveyResultChannel", survey_id: EvawebDashboard.surveyId }, {
-  connected() {
-    // Called when the subscription is ready for use on the server
-    console.log('connected');
+  connected() {},
+  disconnected() {},
 
-  },
+  received(data) {
+    const chartDiv = document.getElementById("chart");
 
-  disconnected() {
-    // Called when the subscription has been terminated by the server
-  },
+    if (chartDiv.classList.contains('col-12')) {
```

Comment:
Hm, I just thought that it probably would be more elegant if you wold add a `data-received: 'true'` attribute